docs/installation/tenants.asciidoc

```diff
@@ -0,0 +1,60 @@
+PacketFence supports multi-tenants and is by default in a mono-tenant mode.
+
+=== General concepts
+
+==== Built-in tenants
+
+Tenant `1` is the default tenant. A default PacketFence installation will use this tenant transparently.
+
+Tenant `0` is the global tenant. In a multi-tenant setup, you can modify
+configuration only in that tenant.
+
+All items configuration are global and share between all tenants. If you want
+to manage a multi-tenant setup, you need to deploy a specific configuration
+that will rely on tenant IDs.
+
+==== Domain name and portal domain name of a tenant
+
+A tenant can be created with two *optional* parameters:
+
+* a domain name: to create users in database using following convention: `username@DOMAIN_NAME_OF_TENANT`
+* a portal domain name: to assign nodes to a specific tenant when they reach PacketFence on this captive portal address
+
+These two parameters are used by code to assign tenants to PacketFence objects.
+
+When possible, we highly recommend you to create tenant with these two parameters.
+
+==== Network devices
```

Comment:
I would put:

Network devices (Switches)

Since that is how they are refered to in the admin